src/components/Post.jsx

```diff
@@ -1,4 +1,14 @@
-const Post = () => {
-  return <div>Post</div>;
+import { Link } from "react-router-dom";
+
+const Post = ({ index }) => {
+  return (
+    <Link className="w-[80%] h-80 border" to={`/post/${index}`}>
+      <div>Post {index}</div>
+    </Link>
+  );
 };
+Post.propTypes = {
```

Comment:
No need to do that since it is javascript

Reply:
eslint sure is strict

Reply:
eslint sure is strict